plugins/linden.py

```diff
@@ -131,7 +131,13 @@ def invest_buy_cmd(user_id, args):
     quote = invest_get_quotes_w_cache([tic])[0]
     if quote['Ask'] is None:
         return "Stock ${} not found (you sure it's on the NASDAQ?)".format(tic)
-    ask = float(quote['Ask'])
+
+    try:
+        ask = float(quote['Ask'])
```

Comment:
This one isn't needed, the API guarantees a float or None and None is checked for